lib/std/tempfile.nim

```diff
@@ -0,0 +1,109 @@
+import std/[os, random]
```

Comment:
rename this module as `std/tempfiles`
> https://nim-lang.github.io/Nim/contributing.html

> New module names should prefer plural form whenever possible, e.g.: std/sums.nim instead of std/sum.nim. In particular, this reduces chances of conflicts between module name and the symbols it defines. Furthermore, module names should use snake_case and not use capital letters, which cause issues when going from an OS without case sensitivity to an OS with it.

Reply:
fixed